rbc/heavydb/remoteheavydb.py

```diff
@@ -1451,7 +1482,6 @@ def normalize_function_type(self, ftype: typesystem.Type):
             # convert `T foo` to `T | name=foo`
             if 'name' not in atype.annotation() and atype.name is not None:
                 atype.annotation(name=atype.name)
-                atype._params.pop('name')
```

Comment:
Why this change?
This will contradict the comment above as the action here is "convert `T foo` to `T foo | name=foo`".

Reply:
Reverted this change